WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -127,6 +127,29 @@ $venue-map-wrapper-height: 400px;
 }
 
 .competitions-table {
+  font-size: 12px;
```

Comment:
As discussed offline, I'm down with decreasing our font size, but I want to do it globally, not like this. Can you find a bootstrap variable to change instead?